libs/wingsdk/src/std/json.ts

```diff
@@ -2,6 +2,98 @@
  * Immutable Json
  */
 export class Json {
+  /**
+   * Returns the keys from the Json object.
+   *
+   * @macro (Object.keys($args$))
+   *
+   * @param json to get keys from
+   * @returns the keys from the Json object as string array
+   */
+  public static keys(json: Json): string[] {
+    json;
+    throw new Error("Macro");
+  }
+
+  /**
+   * Returns the values from the Json.
+   *
+   * @macro (Object.values($args$))
+   *
+   * @param json to get values from
+   * @returns the values from the Json as array of Json
+   */
+  public static values(json: Json): Json[] {
+    json;
+    throw new Error("Macro");
+  }
+
+  /**
+   * Creates a mutable deep clone of the Json.
+   *
+   * @macro (JSON.parse(JSON.stringify($args$)))
+   *
+   * @param json to clone
+   * @returns the mutable clone of the Json
+   */
+  public static cloneMut(json: Json): MutJson {
+    json;
+    throw new Error("Macro");
+  }
+
+  /**
+   * Creates a immutable deep clone of the Json.
+   *
+   * @macro (JSON.parse(JSON.stringify($args$)))
+   *
+   * @param json to clone
+   * @returns the immutable clone of the Json
+   */
+  public static clone(json: Json): Json {
+    json;
+    throw new Error("Macro");
+  }
+
+  /**
+   * Deletes a key in a given Json
+   *
+   * @macro ((args) => { delete (args[0])[args[1]]; })([$args$])
+   *
+   * @param json to delete key from
+   * @param key the key to delete
+   */
+  public static delete(json: Json, key: string): void {
+    json;
+    key;
+    throw new Error("Macro");
+  }
+
+  /**
+   * Formats Json as string
+   *
+   * @macro (JSON.stringify($args$))
+   *
+   * @param json to format as string
+   * @returns string representation of the Json
+   */
+  public static toStr(json: Json): string {
+    json;
+    throw new Error("Macro");
+  }
+
+  /**
+   * Parse a string into a Json
+   *
+   * @macro (JSON.parse($args$))
+   *
+   * @param str to parse as Json
+   * @returns Json representation of the string
+   */
+  public static parse(str: string): Json {
```

Comment:
Still need to implement `.from_json()` methods for our primitive and container types.